src/components/SearchResults.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import TableInfo from "./TableInfo.js";
+import TableRow from "./TableRow.js";
+import CustomerProfile from "./CustomerProfile";
+
+function SearchResults(props) {
+  const [selectedId, setSelectedId] = useState(null);
+  return (
+    <div>
+      <table className="table table-striped">
+        <TableInfo />
+        <tbody>
+          {props.results.map((data, index) => {
+            return (
+              <TableRow key={index} data={data} handleClick={setSelectedId} />
+            );
+          })}
+        </tbody>
+      </table>
+      {selectedId && <CustomerProfile customerId={selectedId} />}
```

Comment:
Well done with storing `selectedId` in state in this component and passing `setSelectedId` function to the `TableRow`! I can see that you have a good understanding of how to connect components with each other in React🙂